gateway/gateway.go

```diff
@@ -40,6 +40,11 @@ type Config struct {
 	// overridden per FQDN in PublicGateways. To be used with WithHostname.
 	NoDNSLink bool
 
+	// RawError disables pretty HTML pages when an error occurs. Instead, a `text/plain`
+	// page will be sent with the raw error message. This can be useful if this gateway
+	// is being proxied by other service, which wants to use the error message.
+	RawError bool
```

Comment:
```suggestion
	// ForcePlainTextErrors disables pretty HTML pages when an error occurs. Instead, a `text/plain`
	// page will be sent with the raw error message. This can be useful if this gateway
	// is being proxied by other service, which wants to use the error message.
	ForcePlainTextErrors bool
```